src/HealthcareApis/Az.HealthcareApis.psd1

```diff
@@ -98,7 +112,7 @@ PrivateData = @{
     PSData = @{
 
         # Tags applied to this module. These help with module discovery in online galleries.
-        Tags = 'Azure','ResourceManager','ARM','HealthCare','HealthcareApis','FhirService'
+        Tags = 'Azure','ResourceManager','ARM','PSModule','HealthcareApis'
```

Comment:
```suggestion
        Tags = 'Azure','ResourceManager','ARM','PSModule','HealthcareApis'
```
Please update your generation PR and add the two missing tags back.
https://github.com/Azure/autorest.powershell/blob/main/docs/metadata.md